lib/libsys/intro.2

```diff
@@ -38,6 +38,7 @@
 .Lb libc
 .Sh SYNOPSIS
 .In errno.h
+.In syscall.h
```

Comment:
```suggestion
.In sys/syscall.h
```
There is not /usr/include/syscall.h.  I'd probably move it above errno.h to follow style(9)